lstchain/image/muon/muon_analysis.py

```diff
@@ -437,7 +437,8 @@ def fill_muon_event(parameters, output_parameters, good_ring, event_id,
                     muonparameters, hg_peak_sample=np.nan, lg_peak_sample=np.nan):
     output_parameters['event_id'].append(event_id)
     output_parameters['event_time'].append(event_time)
-    output_parameters['mc_energy'].append(parameters.mc_energy)
+    if parameters is not None:
```

Comment:
`parameters.mc_energy` should exist also for real data, just filled with `None`
https://github.com/cta-observatory/cta-lstchain/blob/93ccc5c782e89ff1f62e575d266fff085a957fa8/lstchain/io/lstcontainers.py#L75
would that give an error?

Reply:
> `parameters.mc_energy` should exist also for real data, just filled with `None`
> https://github.com/cta-observatory/cta-lstchain/blob/93ccc5c782e89ff1f62e575d266fff085a957fa8/lstchain/io/lstcontainers.py#L75
> 
> 
> would that give an error?

Don't know, but let's move forward.  -1 is a sufficiently clear error value for an energy.